osu.Game/Rulesets/Mods/ModIncognito.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Localisation;
+
+namespace osu.Game.Rulesets.Mods
+{
+    public abstract class ModIncognito : Mod
+    {
+        public override string Name => "Incognito";
+        public override LocalisableString Description => "Obscure some gameplay elements.";
+        public override double ScoreMultiplier => 1;
+        public override string Acronym => "IC";
+        public override IconUsage? Icon => FontAwesome.Solid.UserSecret;
+        public override ModType Type => ModType.Conversion;
```

Comment:
Why is this in `Conversion` again? Completely does not fit the category. Should either be `DifficultyIncrease` or `Fun`.

Reply:
You are right. I will put it at `DifficultyIncrease` section since it aims to always obscure visuals.